src/main/java/de/retest/web/RecheckSeleniumAdapter.java

```diff
@@ -61,18 +63,52 @@ public boolean canCheck( final Object toVerify ) {
 		logger.info( "Retrieving attributes for each element." );
 		final Set<String> cssAttributes = attributesProvider.getCssAttributes();
 		final JavascriptExecutor jsExecutor = (JavascriptExecutor) driver;
-		@SuppressWarnings( "unchecked" ) final Map<String, Map<String, Object>> result =
+		@SuppressWarnings( "unchecked" )
+		final Map<String, Map<String, Object>> rawAttributesMapping =
```

Comment:
If below is `frameAttributesMapping`, maybe this should be `windowAttributesMapping`?